_posts/2023-12-31-end-of-year-update.markdown

```diff
@@ -0,0 +1,150 @@
+
+![Discord](/assets/2023-end-of-year-update/browser-discord.png)
+
+### Hexchat
+_Written by @Dennisbonke_
+
+When we started this project, Discord running was still a pipe dream. Luckily that didn't have to be a big issue, as we mirror our Discord channels to IRC, so an IRC client was needed. At the time, we only had GTK2 available so our choice landed on [`hexchat`](https://hexchat.github.io/). While the porting of `hexchat` went quite smoothly, it sat in review hell for well over a year, forgotten by everyone, including me. While cleaning up PRs, I found it, dusted it off, repaired the bitrot, and got it working again. It was only fair to merge it this time around.
+
+![hexchat](/assets/2023-end-of-year-update/hexchat.png)
+
+### Sway
+_Written by @no92_
+
+[Sway](https://swaywm.org/) is a tiling window manager like i3, but for wayland. After a long period of work, we managed to make it work on managarm.
+
+![Sway](/assets/2023-end-of-year-update/sway.png)
+
+**INSERT SCREENCAP HERE**
+
+Getting managarm up to speed with the features that sway expects and uses took some effort, but are now upstream and will help us in the future. Among those is DRM object sharing via the PRIME ioctls, atomic modesetting and many, many bugfixes.
+
+### htop
+_Written by @Dennisbonke_
+
+With `/proc/self` and `/proc/[pid]` existing in Managarm, having a program to list all processes and do something with that was an obvious addition. So we ported `htop`, which with a few patches was able to list all processes just fine. All in all a fine addition to our collection of ports.
+
+### OpenTTD
+_Written by @ElectrodeYT_
+
+While we already have a few games, none of them were very advanced. OpenTTD, in comparison, is a quite advanced simulation game, and is also open source, with open assets, meaning that it was quite a obvious choice to port.
+
+The only thing that does not work with this port is multiplayer over networks, however as our networking subsystem improves we expect that this will begin working.
+
+![OpenTTD](/assets/2023-end-of-year-update/openttd.png)
+
+### Autoconfig
+_Written by @Dennisbonke_
+
+Last year, we upstreamed our targets into the [config.sub](https://lists.gnu.org/archive/html/config-patches/2022-09/msg00001.html) repo. This year, `autoconf v2.72` was released, which includes our targets! Newly generated autotools tarballs with this version (or newer) will require a lot less patching, ideally none at all, now that we're in upstream autoconf.
```

Comment:
note that autoconf versions do not determine config.sub revisions.  just the default is, and even that is not always the case (if distributors replace the config.sub ac installs)